perfkitbenchmarker/virtual_machine.py

```diff
@@ -192,6 +194,8 @@ def ApplyFlags(self, flags):
       flags.zones.append(self.zone)
     self.machine_type = flags.machine_type or self.machine_type
     self.image = flags.image or self.image
+    self.background_cpu_threads = flags.background_cpu_threads or\
```

Comment:
Not sure if we have a convention, but I find parentheses cleaner for splitting a value across multiple lines.